Source/ASCellNode.h

```diff
@@ -126,10 +125,9 @@ typedef NS_ENUM(NSUInteger, ASCellNodeVisibilityEvent) {
 
 
 /**
- * The owning node (ASCollectionNode/ASTableNode) of this cell node, or @c nil if this node is
- * not a valid item inside a table node or collection node or if those nodes are nil.
+ * The table- or collection-node that this cell is a member of, if any.
  */
-@property (weak, nonatomic, readonly, nullable) ASDisplayNode *owningNode;
+@property (weak, readonly, nullable) id<ASRangeManagedNode> owningNode;
```

Comment:
This s technically a breaking API change but it's so minor that we shouldn't worry about it. Virtually nobody uses this property – it was recently added and probably shouldn't be public anyway – and the migration pathway is trivial.